recipe/build.sh

```diff
@@ -14,13 +14,19 @@ fi
 
 for _shared in OFF ON; do
   mkdir build-${_shared}
-  pushd build-${_shared}
+  pushd build-${_shared} || exit
     cmake ${CMAKE_ARGS}                     \
           -DCMAKE_INSTALL_PREFIX=${PREFIX}  \
           -DBUILD_SHARED_LIBS=${_shared}    \
           -DCMAKE_INSTALL_LIBDIR=lib        \
+          -DCRYPTO_BACKEND=OpenSSL          \
+          -DENABLE_ZLIB_COMPRESSION=ON      \
+          -DBUILD_EXAMPLES=OFF              \
+          -DBUILD_TESTING=OFF               \
```

Comment:
Have enabled tests without docker in a separate PR, let's see how it goes, can explain the changes offline if you would like